dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/event/RumEventSerializerTest.kt

```diff
@@ -478,59 +578,42 @@ internal class RumEventSerializerTest {
 
     // region Internal
 
-    private fun assertSerializedJsonMatchesInputEvent(
-        jsonObject: JsonObject,
-        event: RumEvent
-    ) {
-        assertJsonContainsCustomAttributes(jsonObject, event)
-    }
-
-    private fun assertJsonContainsCustomAttributes(
-        jsonObject: JsonObject,
-        event: RumEvent
-    ) {
-        event.globalAttributes
-            .filter { it.key.isNotBlank() }
-            .forEach {
-                val value = it.value
-                val keyName = "${RumEventSerializer.GLOBAL_ATTRIBUTE_PREFIX}.${it.key}"
-                when (value) {
-                    null -> assertThat(jsonObject).hasNullField(keyName)
-                    is Boolean -> assertThat(jsonObject).hasField(keyName, value)
-                    is Int -> assertThat(jsonObject).hasField(keyName, value)
-                    is Long -> assertThat(jsonObject).hasField(keyName, value)
-                    is Float -> assertThat(jsonObject).hasField(keyName, value)
-                    is Double -> assertThat(jsonObject).hasField(keyName, value)
-                    is String -> assertThat(jsonObject).hasField(keyName, value)
-                    is Date -> assertThat(jsonObject).hasField(keyName, value.time)
-                    is JsonObject -> assertThat(jsonObject).hasField(keyName, value)
-                    is JsonArray -> assertThat(jsonObject).hasField(keyName, value)
-                    is Iterable<*> -> assertThat(jsonObject).hasField(keyName, value.toJsonArray())
-                    is Map<*, *> -> assertThat(jsonObject).hasField(keyName, value.toJsonObject())
-                    else -> assertThat(jsonObject).hasField(keyName, value.toString())
-                }
+    private fun Forge.forgeRumEvent(
+        attributes: Map<String, Any?> = emptyMap(),
+        userAttributes: Map<String, Any?> = emptyMap()
+    ): Any {
+        return when (this.anInt(min = 0, max = 5)) {
```

Comment:
`aValueFrom` can be used instead of Int.

Reply:
not sure I understand. And how would you apply it ?

Reply:
my bad, I mistyped the method. I meant `anElementFrom(ViewEvent::class.java, ....)`. But works fine for me like that as well.